lib/services/types/java.service.js

```diff
@@ -0,0 +1,68 @@
+const { forOwn } = require('lodash')
+
+/**
+ * Create an string of a Java class
+ * @param {*} model the sequelize model
+ * @param {*} modelname the name of the model 
+ * @returns the class to write in the file
+ */
+function buildJava(model, modelname) {
```

Comment:
I'll add a new issue to create an empty constructor, getters and setters. What is the point of the exported class if you can't use it?